src/core_modules/capture-core/components/Pages/MainPage/EventWorkingLists/epics/updateEventWorkingList.js

```diff
@@ -23,16 +23,20 @@ export const updateEventWorkingListAsync = (
     columnsMetaForDataFetching: ColumnsMetaForDataFetching,
     categoryCombinationMeta: Object,
     storeId: string,
-}): Promise<ReduxAction<any, any>> => getEventWorkingListDataAsync(
+}): Promise<ReduxAction<any, any>> => getEventListData(
     buildQueryArgs(
         queryArgsSource, {
             columnsMetaForDataFetching,
             storeId,
             isInit: false,
         }),
     columnsMetaForDataFetching, categoryCombinationMeta)
-    .then(data =>
-        updateListSuccess(storeId, data),
+    .then(({ eventContainers, pagingData, request }) =>
+        updateListSuccess(storeId, {
+            recordContainers: eventContainers,
+            pagingData,
+            request,
+        }),
```

Comment:
That's an example of what I was saying on those `...args` dispatch functions. Now it is far more clear what you are passing around.